tests/release/const.go

```diff
@@ -0,0 +1,44 @@
+package constants
+
+import (
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	ApplicationNameDefault          string = "appstudio"
+	ComponentName                   string = "dc-metro-map"
+	GitSourceComponentUrl           string = "https://github.com/scoheb/dc-metro-map"
+	AdditionalComponentName         string = "simple-python"
+	AdditionalGitSourceComponentUrl string = "https://github.com/devfile-samples/devfile-sample-python-basic"
```

Comment:
I am fine with one constant file, but if we go with three (this one, the service specific one, the pipelines specific one), this would be a pipelines specific variable